modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/bean/MigrateState.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
the bean package can be moved to the "models" module. Thats where all data models are present.